components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -3856,4 +3856,19 @@ private static void endTenantFlow() {
 
         PrivilegedCarbonContext.endTenantFlow();
     }
+
+    /**
+     * Determines if the scope is specified in the whitelist.
+     *
+     * @param scope - The scope key to check.
```

Comment:
Missing param in javadoc comment

Reply:
fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/9f55ee3aa001b5471ee20f6625ffebee8394af5e